src/view/observer.js

```diff
@@ -313,10 +313,13 @@ Candy.View.Observer = (function(self, $) {
 	 *
 	 * Parameters:
 	 *   (jQuery.Event) event - jQuery Event object
-	 *   (Object) args - {presetJid}
+	 *   (Object) args - {messageKey, presetJid}
 	 */
 	self.Login = function(event, args) {
-		Candy.View.Pane.Chat.Modal.showLoginForm(null, args.presetJid);
+		args = args || {};
+		args.messageKey = args.messageKey || '';
+		args.presetJid = args.presetJid || null;
```

Comment:
@billybennett we do code reviews to review code. If we don't agree with something, we tell it to you. Don't take that personally.

I agree with @benlangfeld that the default assignments are not necessary. If they don't get passed, they're `either` null or `undefined`.